policyengine_us/parameters/gov/states/nm/tax/income/credits/working_families_tax/match.yaml

```diff
@@ -0,0 +1,10 @@
+description: New Mexico matches the federal EITC at this rate.
+values:
+  2021-01-01: 0.2
+  2023-01-01: 0.25
+metadata:
+  unit: /1
+  label: New Mexico working families tax credit match
+  reference:
+    - title: New Mexico Income Tax Act, 7-2-18.15, A
```

Comment:
Cite the legal code instead:
https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!b/7-2-18.15

As well as the latest tax form